info-compiler/src/test/java/com/google/sps/InfoCompilerTest.java

```diff
@@ -14,6 +14,9 @@
 
 package com.google.sps.infocompiler;
 
+import static com.google.common.truth.Truth.*;
+import static com.google.common.truth.Truth8.*;
+import static org.mockito.Mockito.*;
 import com.google.cloud.datastore.BooleanValue;
```

Comment:
nit: single blank line separating static and non-static imports
```suggestion
import static org.mockito.Mockito.*;

import com.google.cloud.datastore.BooleanValue;
```
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing